fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/AppAndSystemHelper.kt

```diff
@@ -139,16 +143,21 @@ object AppAndSystemHelper {
 
     fun assertExternalAppOpens(appPackageName: String) {
         if (isPackageInstalled(appPackageName)) {
+            Log.i(TAG, "assertExternalAppOpens: $appPackageName is installed on device")
             try {
-                Intents.intended(IntentMatchers.toPackage(appPackageName))
+                Log.i(TAG, "assertExternalAppOpens: Try block")
+                intended(toPackage(appPackageName))
+                Log.i(TAG, "assertExternalAppOpens: Matched intent to $appPackageName")
```

Comment:
@jjSDET @AndiAJ 

This seems a bit excessive. How about simply logging to the exception when thrown? Example in `isPackageInstalled` https://github.com/mozilla-mobile/firefox-android/blob/d89c4db8a431fc06feb74024dc40e60f6f9a2ac7/fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/AppAndSystemHelper.kt#L136 let's do the same thing as below

`Log.i(TAG, "assertExternalAppOpens: isPackageInstalled - ${e.message}")` this tells us that the exception was thrown and the package was not installed